src/Apps/Collect/__tests__/routes.test.tsx

```diff
@@ -0,0 +1,170 @@
+import { mount } from "enzyme"
+import { Resolver } from "found-relay"
+import createRender from "found/lib/createRender"
+import getFarceResult from "found/lib/server/getFarceResult"
+import { find } from "lodash"
+import { Environment, RecordSource, Store } from "relay-runtime"
+import { Boot } from "../../../Artsy/Router/Components/Boot"
+import {
+  createMockNetworkLayer,
+  MockBoot,
+  renderUntil,
+} from "../../../DevTools"
+import { routes } from "../routes"
+
+describe("Routes", () => {
+  async function render(url, mockResolvers) {
+    const network = createMockNetworkLayer(mockResolvers)
+    const source = new RecordSource()
+    const store = new Store(source)
+    const environment = new Environment({ network, store })
+
+    // return await getFarceResult({
+    //   url,
+    //   routeConfig: routes,
+    //   resolver: new Resolver(environment),
+    //   render: createRender({}),
+    // })
+
+    return (
+      <Boot
+        context={{}}
+        user={{}}
+        relayEnvironment={environment}
+        resolver={new Resolver(environment)}
+        routes={routes}
+      />
+    )
+  }
+
+  describe("#/collect/:medium?", () => {
+    let route = null
+    const params = {}
+
+    beforeEach(() => {
+      route = find(routes, ["path", "/collect/:medium?"])
+    })
+
+    it("sends correct default variables for query", () => {
+      expect(route.prepareVariables(params, {})).toEqual({
+        sort: "-decayed_merch",
+      })
+    })
+
+    xit("renders", async () => {
```

Comment:
still working on getting this test to work. It's a little tricky as it's attempting to test the router as well 
but I may just drop down to testing only the `CollectApp` and the relay container instead.

To get the fixes in, I'm commenting it out for now but will fix it in another PR